scripts/install-worker.sh

```diff
@@ -125,7 +126,7 @@ if [[ "$INSTALL_DOCKER" == "true" ]]; then
     # Due to an issue with the latest containerd, customers are seeing
     # pods getting stuck in terminating, so we need to downgrade containerd
     # until the issue is fixed upstream or we have another workaround.
-    sudo yum downgrade -y containerd-1.3.2-1.amzn2
+    sudo yum downgrade -y containerd-${CONTAINERD_VERSION}
```

Comment:
Should this be removed or changed to use `yum install`?

Reply:
`yum install` will fail if we're trying to install a version that older than the current version installed (containerd is installed as a dependency on docker).

`yum downgrade` won't fail if we're a) downgrading to an older version or b) downgrading to the current version that's already installed when we installed docker.